### PR TITLE
docs(redis): Update externalize-redis.md

### PR DESCRIPTION
**Pull request by celamb4**
Add custom gate configuration required with external redis.
https://github.com/spinnaker/spinnaker.github.io/issues/986

### CODE REVIEW

File: setup/productionize/caching/externalize-redis.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ overrideBaseUrl: $REDIS_ENDPOINT
 skipLifeCycleManagement: true
 ```
 
+Additionally, we will create the following file in order to customize the gate service. `~/.hal/$DEPLOYMENT/profiles/gate-local.yml`:
```
Comment (avh121) [inline setup/productionize/caching/externalize-redis.md]: nit: replace "we will create..." with "you'll need to create..."

File: setup/productionize/caching/externalize-redis.md [hunk 2]
```diff
@@ -38,6 +38,14 @@ overrideBaseUrl: $REDIS_ENDPOINT
...
+```yaml
+redis:
+    configuration:
+         secure:
+              true
```
Comment (lwander) [inline setup/productionize/caching/externalize-redis.md:46]: I don't think this needs a new line

**Review by lwander — APPROVED** [review]
Thanks!

File: setup/productionize/caching/externalize-redis.md (hunk 1 above)
Comment (dorbin) [reply setup/productionize/caching/externalize-redis.md]: or just: "Create..."

**Review by dorbin — CHANGES_REQUESTED** [review]
Hi, @celamb4 , thanks for the update. Can you address the comments on the wording, and I'll go ahead and merge this?

**Comment by seanpeters86** [comment]
@celamb4 any update here?

**Review by 317brian — APPROVED** [review]
Made the requested change. Merging!

**Pull request merged**